front/app/components/admin/GraphCards/ActiveUsersCard/Chart.tsx

```diff
@@ -67,7 +66,7 @@ const Chart = ({
     return null;
   }
 
-  const noData = isNilOrError(timeSeries);
+  const noData = !!timeSeries;
```

Comment:
Should it be like that?
```suggestion
  const noData = !timeSeries;
```

Reply:
Nice catch